Challange-1-iOS/Challange-1-iOS/ViewControllers/AppleReposViewController.swift

```diff
@@ -7,14 +7,158 @@
 
 import UIKit
 
+struct ConstantsAppleRepos {
+    static let rowHeight: CGFloat = 50
+}
+
 class AppleReposViewController: UIViewController, Coordinating {
+    private var tableView: UITableView
+    private var appleRepos: [AppleRepos] = []
+    private var mockedAppleReposDataSource: MockedAppleReposDataSource = .init()
```

Comment:
ideally the mocked object should not be on the class, you should inject it from outside only when it's necessary (during the tests)

Reply:
@kevenm4 @renatasiqueira @RubenAntunes16 ^

Reply:
@cpradobliss So I should create the var mockedAppleReposDataSource and make it optional? and when I want, in coordinator, I create the instance and inject in the AppleReposViewController?